README-zephyr.rst

```diff
@@ -31,16 +31,18 @@ Building the bootloader itself
 
 The bootloader is an ordinary Zephyr application, at least from
 Zephyr's point of view.  There is a bit of configuration that needs to
-be made before building it.  Most of this is done in the top-level
-``Makefile`` in the source tree.  There are comments there for
+be made before building it.  Most of this can be done as documented in
+the ``CMakeLists.txt`` file in boot/zephyr.  There are comments there for
 guidance.  It is important to select a signature algorithm, and decide
 if slot0 should be validated on every boot.
 
-There is a ``build_boot.sh`` script at the top level that can make
-building a bit easier.  It assumes that the mcuboot tree is next to,
-at the same level, as the zephyr source tree.  It takes a single
-argument, which is the target to build. This must be a Zephyr board
-which supports mcuboot.
+To build mcuboot, create a build directory in boot/zephyr, and build
+it as usual:
+
+$ cd boot/zephyr
+$ mkdir build && cd build
+$ cmake -DBOARD=your_board ..
```

Comment:
for consistency with the existing Zephyr doc I would recommend using `-DBOARD=<board>`